[2023-02-19T13:42:53.464+0200] {taskinstance.py:1161} INFO - Dependencies not met for <TaskInstance: test_process_sales2.format_data_to_avro scheduled__2023-02-08T03:44:00+00:00 [failed]>, dependency 'Task Instance State' FAILED: Task is in the 'failed' state.
[2023-02-19T13:42:53.467+0200] {local_task_job.py:101} INFO - Task is not able to be run
